apps/staging-enterprise/__tests__/integration

```diff
@@ -5,7 +5,6 @@ set -e
 cd "`dirname "$0"`/.."
 
 # Set needed ENV
-export GROUPAROO_TELEMETRY_ENABLED=false
```

Comment:
Why remove these from the bash tests?

Reply:
although, I guess everyone running tests locally should have telemetry disabled already in their env?

Reply:
Yeah - we all should have telemetry disabled.  And you shouldn't need to mess with the tests to re-record the nocks - just change your ENV.

I was most concerned with our developer experience going forward.  You shouldn't need to remember to change your environment in any future tests; the OS/platform should handle this for you.